var/spack/repos/builtin/packages/postgis/package.py

```diff
@@ -0,0 +1,79 @@
+# Copyright 2013-2019 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class Postgis(AutotoolsPackage):
+    """
+    PostGIS is a spatial database extender for PostgreSQL object-relational
+    database. It adds support for geographic objects allowing location
+    queries to be run in SQL
+    """
+
+    homepage = "https://postgis.net/"
+    url      = "https://download.osgeo.org/postgis/source/postgis-2.5.3.tar.gz"
+
+    version('3.0.1', sha256='5a5432f95150d9bae9215c6d1c7bb354e060482a7c379daa9b8384e1d03e6353')
+    version('3.0.0', sha256='c06fd2cd5cea0119106ffe17a7235d893c2bbe6f4b63c8617c767630973ba594')
+    version('2.5.3', sha256='72e8269d40f981e22fb2b78d3ff292338e69a4f5166e481a77b015e1d34e559a')
+
+    variant('gui', default=False, description='Build with GUI support, creating shp2pgsql-gui graphical interface to shp2pgsql')
+
+    # Refs:
+    # https://postgis.net/docs/postgis_installation.html
+    # https://postgis.net/source/
+
+    depends_on('postgresql')
+    depends_on('geos')
+    depends_on('proj')
```

Comment:
I'm a little nervous to see a PROJ dependency without version requirements, since they keep changing their interface. Does this really work with _any_ version of PROJ?

Reply:
Not sure. I am not a user of this package. Guix and homebrew are also building it with latest proj. In any case, proj version will be constrained by gdal build.

https://formulae.brew.sh/formula/postgis

Reply:
I wonder if @citibeth might have an opinion about this. Hope she doesn't mind us pinging her.